tests/unit_tests/test_cli.py

```diff
@@ -346,14 +447,39 @@ def test_invalid_stomp_config_for_listener(runner: CliRunner):
 
 
 def test_cannot_run_plans_without_stomp_config(runner: CliRunner):
-    result = runner.invoke(main, ["controller", "run", "sleep", '{"time": 5}'])
+    result = runner.invoke(
+        main,
+        [
+            "controller",
+            "run",
+            "sleep",
+            "-i",
+            "cm12345-1",
+            '{"time": 5}',
+        ],
+    )
     assert result.exit_code == 1
     assert (
         result.stderr
         == "Error: Stomp configuration required to run plans is missing or disabled\n"
     )
 
 
+def test_cannot_start_a_plan_without_an_instrument_session(runner: CliRunner):
+    result = runner.invoke(
+        main,
+        [
+            "controller",
+            "run",
+            "--background",
+            "sleep",
+            '{"time": 5}',
+        ],
+    )
+    assert result.exit_code == 1
```

Comment:
This test fails if the user has a visit cached. Probably want to mock some things out. Possibly with an autouse fixture to block the user's real cache being used in any of the tests.